dashboards/magma-cluster.json

```diff
@@ -0,0 +1,211 @@
+{
+    "_base": "dashboard",
+    "title": "Magma - Cluster",
+    "_panels": [
+        {
+            "_base": "panel",
+            "title": "Gets rate",
+            "gridPos": {
+                "h": 9
+            },
+            "_targets": [
+                {
+                    "datasource": "{data-source-name}",
+                    "expr": "rate(kv_ep_magma_gets[20s])",
```

Comment:
Hey Rohan - `kv_ep_magma_gets`is a per-bucket stat - so you should include that in the legend (i.e. something like `{data-source-name}:{{bucket}}`. Perhaps better is to sum across all buckets since this is a cluster view - and if you have a lot of nodes and a lot of buckets the panel will be impossible to read. 

Same comment for the rest of the stats on this dashboard.

Reply:
Thanks for catching that. Certainly my intention was to aggregate across all buckets. I missed using the `sum` function. I'll fix it.